projects/Amlogic/bootloader/mkimage

```diff
@@ -3,28 +3,61 @@
 
 . config/options $1
```

Comment:
```suggestion
. config/options ""
```
as arg1 will be the arg1 passed to `scripts/mkimage ` (currently nothing, but probably not what you want here).